task_types.go

```diff
@@ -43,10 +43,6 @@ func defaultTaskIngestionSpec() *TaskIngestionSpec {
 		Spec: &IngestionSpecData{
 			DataSchema: &DataSchema{
 				DataSource: "some_datasource",
```

Comment:
IMHO, this can also be confusing. What about `"unspecified"` or something similar?